dtp_apis/link_DTP_API.py

```diff
@@ -250,3 +250,41 @@ def link_node_constr_to_operation(self, constr_node_iri, oper_node_iri):
                 logger_global.error("Linking nodes failed. Response code: " + str(response.status_code))
                 return False
         return True
+
+    def link_node_asperf_to_asplanned(self, asperf_iri, asplannned_iri):
+        """
+        The method links an as-performed to as-planned node
+
+        Parameters
+        ----------
+        asperf_iri : str, obligatory
+            a valid IRI of an as-performed node
+        asplannned_iri : str, obligatory
+            a valid IRI of an as-planned node
+
+        Returns
+        ------
+        bool
+            True if the nodes has been linked, and False otherwise
+        """
+
+        payload = json.dumps([{
+            "_domain": self.DTP_CONFIG.get_domain(),
+            "_iri": asperf_iri,
+            "_outE": [{
+                "_label": self.DTP_CONFIG.get_ontology_uri('intentStatusRelation'),
+                "_targetIRI": asplannned_iri
+            }]
```

Comment:
currently all `intentStatusRelation` links are from as-performed to as-planned nodes.

Reply:
@alwynmathew @jschlenger 

Maybe we could make it more general. Instead of saying link as-performed and as-planned, we can just say link A to B via `intentStatusRelation`.